q.cpp

```diff
@@ -0,0 +1,245 @@
+#include <stdio.h>
+#include <string.h>
+#include <malloc.h>
```

Comment:
`q` wasn't a target in `makefile` before this PR (or it was?) and `makefile` is not in the list of modified files of this PR. Thus, how can this be possible? :o: